contracts/src/CrossChainApplications/examples/NativeTokenBridge/tests/NativeTokenDestinationTests.t.sol

```diff
@@ -94,6 +102,7 @@ contract NativeTokenDestinationTest is Test {
             _DEFAULT_OTHER_BRIDGE_ADDRESS,
             abi.encode(_DEFAULT_RECIPIENT, _DEFAULT_INITIAL_RESERVE_IMBALANCE)
         );
+        vm.stopPrank();
```

Comment:
Does `vm.stopPrank` need to be called for every `vm.prank`?

Reply:
no it does not, was not meant to be pushed, thanks for catching